controllers/grid/users/reviewer/form/ReviewReminderForm.inc.php

```diff
@@ -141,15 +144,26 @@ function execute($args, $request) {
 		$user = $request->getUser();
 
 		import('lib.pkp.classes.mail.SubmissionMailTemplate');
-		$email = new SubmissionMailTemplate($submission, 'REVIEW_REMIND', null, null, null, false);
+		$context = $request->getContext();
+		$templateKey = $this->_getMailTemplateKey($context);	
+		$email = new SubmissionMailTemplate($submission, $templateKey, null, null, null, false);
+		
+		$reviewUrlArgs = array('submissionId' => $reviewAssignment->getSubmissionId());
+		if ($context->getSetting('reviewerAccessKeysEnabled')) {
+			import('lib.pkp.classes.security.AccessKeyManager');
+			$accessKeyManager = new AccessKeyManager();
+			$expiryDays = $context->getSetting('numWeeksPerReview') + 4 * 7;
```

Comment:
@ajnyga, would you mind adding the missing parentheses here too? Should be:

    $expiryDays = ($context->getSetting('numWeeksPerReview') + 4) * 7;